engine/interpreter/src/test/scala/pl/touk/nussknacker/engine/marshall/ProcessMarshallerSpec.scala

```diff
@@ -39,10 +72,7 @@ class ProcessMarshallerSpec extends FlatSpec with Matchers with OptionValues wit
         .enricher("d", "barVar", "dService", "p1" -> "expr3")
         .switch("f", "expr4", "eVar", nestedGraph("e"), Case("e1", GraphBuilder.emptySink("endE1", "")))
 
-    val marshalled = ProcessMarshaller.toJson(process, PrettyParams.spaces2)
-
-    val unmarshalled = ProcessMarshaller.fromJson(marshalled).toOption
-    val result = ProcessCanonizer.uncanonize(unmarshalled.value).toOption
+    val result = marshallAndUnmarshall(process)
```

Comment:
Use `marshallAndUnmarshall` helper in other tests too.

Reply:
Done.